content/release-10-17-0/digital-twin-manager-10-17-0-bundle/10-17-0-0-696.md

```diff
@@ -0,0 +1,53 @@
+---
+weight: 40
+title: Release 10.17.0.0.696
+layout: bundle
+---
+
+### Improvements
+
+<div><table ><colgroup>
+<col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Role-based access to DTM entities has been implemented. </td>
+<td>CTM-649</td>
+</tr>
+
+</tbody></table></div>
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Handled the run-time exceptions during asset creation.</td>
+<td>CTM-687</td>
+</tr>
+
+<tr>
+<td>Handled the run-time exceptions for asset models and asset properties.</td>
+<td>CTM-692</td>
+</tr>
+
+<tr>
+<td>Missing German translations have been added for DTM pages.</td>
```

Comment:
@spau6 What do you mean with pages?

Reply:
Screens like Assets, Asset models, Asset Properties.